opennms-services/src/main/java/org/opennms/netmgt/notifd/ServiceRegistryExecutor.java

```diff
@@ -28,25 +28,28 @@
 
 package org.opennms.netmgt.notifd;
 
+import java.lang.management.ManagementFactory;
+import java.util.List;
+
+import org.opennms.core.soa.lookup.ServiceLookup;
+import org.opennms.core.soa.lookup.ServiceLookupBuilder;
 import org.opennms.core.soa.support.DefaultServiceRegistry;
 import org.opennms.netmgt.model.notifd.Argument;
 import org.opennms.netmgt.model.notifd.NotificationStrategy;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.lang.management.ManagementFactory;
-import java.util.List;
-import java.util.concurrent.TimeUnit;
-
 public class ServiceRegistryExecutor implements ExecutorStrategy {
     private static final Logger LOG = LoggerFactory.getLogger(ServiceRegistryExecutor.class);
 
-    private static final DefaultServiceRegistry s_registry = DefaultServiceRegistry.INSTANCE;
-
     private static final String GRACE_PERIOD_MS_SYS_PROP = "org.opennms.netmgt.notifd.notificationStrategyGracePeriodMs";
```

Comment:
Can we move these properties and default into the service lookup classes? I think we'd be better off having a global properties instead of having to tune these for every interface.

I'm thinking something along the lines of  `org.opennms.core.soa.lookup.gracePeriodMs` and `org.opennms.core.soa.lookup.lookupDelayMs`.

Reply:
Yep, makes sense.